newIDE/app/src/EventsFunctionsExtensionEditor/ObjectMethodSelectorDialog.js

```diff
@@ -156,14 +157,13 @@ export default function ObjectMethodSelectorDialog({
         />
         <MethodListItem
           icon={<Step style={styles.icon} />}
-          name={'doStepPreEvents'}
-          disabled={eventsFunctions.hasEventsFunctionNamed('doStepPreEvents')}
+          name={'doStepPostEvents'}
```

Comment:
What's the reason to avoid doStepPreEvents (sorry maybe we discussed that already)?

Reply:
It's easier to add one that remove one so I prefer to keep it hidden until we find a use-case for it.
Object events are mainly there to update the rendering according to scene or parent event changes so it might not be necessary to run events before the scene events.

Reply:
Fine with this 👍